grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -94,11 +94,11 @@ class DocumentTemplateService {
     }
 
     private createOrderContext(IXDocReport report, Order orderInstance) {
-
-        // Add data to the context
-        def orderItems = orderInstance?.orderItems?.findAll {
+        def activeOrderItems = orderInstance?.orderItems?.findAll {
             it.orderItemStatusCode != OrderItemStatusCode.CANCELED
-        }?.collect { OrderItem orderItem ->
+        }
+        // Add data to the context
+        def orderItems = activeOrderItems?.collect { OrderItem orderItem ->
```

Comment:
Why was this change needed? I think the code was more readable with the chaining.